docs/start/index.rst

```diff
@@ -11,48 +11,36 @@
     useful_helpers/index.rst
     products/index.rst
     animations/index.rst
+    glossary.rst
     changelog.rst
 
 Photons
 =======
 
-Photons is an async Python3.6+ framework for interacting with LIFX devices.
+Photons is an asynchronous Python3.6+ framework for interacting with LIFX devices.
 
-Photons gives you:
+Photons provides:
 
 * High level creation of LIFX :ref:`packets <packets>`
 * Transparent device :ref:`discovery <sender_discovery>`
-* Convenient API for :ref:`sending <sender_interface>` packets to multiple
+* A convenient API for :ref:`sending <sender_interface>` packets to multiple
   devices in parallel and streams responses back.
 * A :ref:`gatherer <gatherer_interface>` for sending the least number of
   messages to get information from devices.
 * A :ref:`product registry <products>` for determining the capabilities of the
   devices on your network.
-* Efficient :ref:`tile animations <animations_root>` that can even run on
-  a Raspberry Pi
-* A rich set of functionality you an access from the
+* Efficient :ref:`tile animations <animations_root>` that can run on
+  low powered devices like a Raspberry Pi
+* A rich set of functionality that can be accessed from the
   :ref:`command line <commandline_root>`
-* The ability to :ref:`configure <configuration_root>` Photons.
 * A :ref:`daemon <device_finder>` you can run for continuous discovery and
   information gathering on your network.
 * Useful :ref:`functionality <useful_helpers_root>` for common tasks while
   creating scripts.
-
-Installation
```

Comment:
I included this here as well as the installation page so that it's on the first page you see. I like when libraries do this to make it as easy as possible to see where to get it from

Reply:
and on the installation page, so that in other pages you can see it in the sidebar